iot/models/iot_device.py

```diff
@@ -13,6 +13,9 @@ class IoTDevice(models.Model):
         'iot.device.action',
         inverse_name='device_id'
     )
+    active = fields.Boolean(default=True)
+    device_ident = fields.Char(required=False)
```

Comment:
if not required, the parameter is not needed.

Also, shouldn't this be called `device_identification`? Not a big issue for me, but this name is somehow strange

Reply:
Ideally this would be device_id, but this will clash with naming convention for many2one in Odoo ORM.
I was trying to keep a small length, but I will change it.

Regarding active, should I make it be required?